CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 CHANGELOG
```

Comment:
minor: not related to this PR, could you fix dependency github action before merge? For example in develop, https://github.com/aws/aws-parallelcluster/pull/3284